gpu4pyscf/tdscf/tests/test_tduks.py

```diff
@@ -171,7 +171,7 @@ def test_tda_m06l(self):
         es = td.kernel(nstates=5)[0]
         ref = td.to_cpu().kernel(nstates=5)[0]
         self.assertAlmostEqual(abs(es - ref[:5]).max(), 0, 8)
-        self.assertAlmostEqual(lib.fp(es), -0.7531765338077471, 6)
+        self.assertAlmostEqual(lib.fp(es), -0.7530329968766932, 6)
```

Comment:
Why are those values changed?

Reply:
Because the test was created using direct scf ground state, computed without density fitting. They are changed to density fitting here.